x/sequencer/genesis.go

```diff
@@ -10,6 +10,15 @@ import (
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
+	// Validate all other genesis fields are empty
+	genesisFields := []int{
```

Comment:
can we just do 
`if len(genState.SequencerList) != 0 {
        return fmt.Errorf("invalid genesis state: SequencerList should be empty")
   }`

Reply:
it's in the init genesis, which doesn't except error to be retuned, but panicing on failure.

Reply:
Then we can panic instead of returning error maybe

Reply:
ha you mean that genesisFields is not needed because it's one field. gotcha.